api/src/pcapi/tasks/batch_tasks.py

```diff
@@ -57,3 +63,10 @@ def track_event_task(payload: TrackBatchEventRequest) -> None:
     push_notifications.track_event(
         payload.user_id, payload.event_name, payload.event_payload, can_be_asynchronously_retried=True
     )
+
+
+@task(settings.GCP_BATCH_CUSTOM_EVENT_QUEUE_NAME, "/batch/track_event/bulk")
+def track_event_for_multiple_users_task(payload: BulkTrackBatchEventRequest) -> None:
```

Comment:
Je vois pas trop l'intérêt d'utiliser une cloud task pour cette fonctionnalité. Ça tourne dans un cron donc ce n'est pas grave si l'appel est synchrone, si ? De plus il n'y a qu'un appel (ou alors plusieurs si batch a une limite sur le nombre d'utilisateurs max qu'il peut traiter ?)

Reply:
1 seul appel effectivement

Reply:
Je vais garder la cloud task parce que je vais en avoir besoin pour d'autres events, mais dans ce cas je vais remplacer
```suggestion
- batch_tasks.track_event_for_multiple_users_task.delay(
+ batch_tasks.track_event_for_multiple_users_task(
```